js/views/notificationsPanelVw.js

```diff
@@ -0,0 +1,106 @@
+var __ = require('underscore'),
+    Backbone = require('backbone'),
+    $ = require('jquery'),
+    loadTemplate = require('../utils/loadTemplate'),
+    notificationsCollection = require('../collections/notificationsCl'),
+    notificationView = require('./notificationVw'),
+    notifications = require('../models/notificationsMd');
+
+module.exports = Backbone.View.extend({
+
+  el: '#notificationsPanel',
+
+  initialize: function(options){
+    var self = this;
+    this.options = options || {};
+    this.parentEl = $(options.parentEl);
+    this.listWrapper = $('<div class="border0 custCol-border-secondary flexRow"></div>');
+
+    this.notificationLimit = 30;
+    this.notifications = new notificationsCollection();
+    this.notifications.url = options.serverUrl + "get_notifications";
+    this.notifications.fetch({
+      success: function(notifications, response) {
+        var unread_count = 0;
+        __.each(notifications.models, function(notification){
+          "use strict";
+          if(notification.get('read') != true) {
+            unread_count = parseInt(unread_count) + 1;
```

Comment:
You should be able to just do unread_count++ here, is there a situation where unread_count gets converted to NaN?